src/lib/selections.ts

```diff
@@ -491,6 +491,11 @@ function resetAndSetEngineEntitySelectionCmds(
   ]
 }
 
+/**
+ * @deprecated was made for guards on modeling machine.
+ * Now the selection command bar argument input handles this.
+ * Leaving until we're sure it's not needed elsewhere.
+ */
```

Comment:
These will never get clean up unless you delete them now, we have the PR if we ever need to put them back.

Reply:
The way I check if an exported function is needed is to delete it, then `yarn tsc`, no errors that we're all good.

Reply:
Cool trick I'll use it!

Reply:
Good call, I realized that one of them is used within other functions 😅. Leaving that one and removing the other.